compiler/one-cmds/onnx_legalizer.py

```diff
@@ -21,12 +21,27 @@
 import re
 
 # Transform onnx model to make it compilable with our toolchain
+#
+# This code works with onnx model in proto format. See proto buffers format in
+# https://github.com/onnx/onnx/blob/96516aecd4c110b0ac57eba08ac236ebf7205728/onnx/onnx.proto3
+#
+# More examples of handling onnx models could be found here:
+# https://github.com/onnx/onnx/tree/96516aecd4c110b0ac57eba08ac236ebf7205728/onnx/examples
+#
 # List of transformations:
 # - Replace RNN operation with unrolled subgraph
 # - Replace LSTM operation with unrolled subgraph
 
 
 class LegalizeOptions:
+    """Determines actions to perform be legalizer
```

Comment:
`Determines actions to perform legalizer` ?

Reply:
I tried to reword this sentence =)